.github/workflows/auto-approve-dependabot.yml

```diff
@@ -0,0 +1,44 @@
+# NOTE: this is a trial to auto-approve PRs from dependabot
+# Reference: https://nicolasiensen.github.io/2022-07-23-automating-dependency-updates-with-dependabot-github-auto-merge-and-github-actions/
+
+name: Dependabot reviewer
+
+on: pull_request_target
+
+permissions:
+  pull-requests: write
+  contents: write
+
+jobs:
+  review-dependabot-pr:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.pull_request.user.login == 'dependabot[bot]' }}
+    steps:
+      - name: Dependabot metadata
+        id: dependabot-metadata
+        uses: dependabot/fetch-metadata@v1
+      - name: Enable auto-merge for Dependabot PRs
+        run: gh pr merge --auto --merge "$PR_URL"
```

Comment:
Question out of curiosity: Why are you using the gh cli? Because it is mighty? Because it works especially well in github actions? Personal preference?